### DISCUSSION TITLE
Need help understanding how to view project

### DISCUSSION THREAD

**Discussion by lisach8714** [community/community#42664] [New to GitHub]
I Am brand new to github. I've been searching for a specific kind of program to view images on a local Drive using a kind of flip book GUI. In my search I found a Python project on GitHub that appears to be what I'm looking for.

[Flipbook for viewing local images](https://github.com/broadinstitute/flipbook) 

 I don't have programming experience and I'm not interested in installing programming language. I just want to know if this is what I'm looking for. I can find no way of asking the user if it is indeed what I'm looking for. Is there a simple way to see what the project actually is, and does? I read somewhere about cloning the desktop.

If it actually does what I think it does then I can move forward and get necessary code to run it.

**Comment by jfullstackdev** [comment]
you can simply look at the README file and I think there is the good details 

![image](https://user-images.githubusercontent.com/47092464/209563481-683a57e0-ff6c-4ed0-a11e-e945babca6fe.png)

**Reply by lisach8714** [reply]
Well yes I did thoroughly read the files that I could read including this one. But I want to see it. Its important that I see how it looks. I'm trying to find a simple GUI that I can run on a touchscreen windows tablet. One that I can configure for my elderly mother to view locally stored photos which replicate from a cloud based photo sharing platform. A GUI that would operate as a flipbook, very simplistic, much like for a 3 year old. I need to see how it looks.
